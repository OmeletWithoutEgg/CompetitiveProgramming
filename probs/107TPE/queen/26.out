-157987.21 837671.99
