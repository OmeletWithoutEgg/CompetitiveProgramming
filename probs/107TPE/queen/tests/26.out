168958.54 -3440169.39
